iOS Client/PodcastsController/PodcastFilterViewController.swift

```diff
@@ -24,6 +29,7 @@ class PodcastsFilterViewController: UIViewController, UITableViewDataSource, UIT
         tableView.estimatedRowHeight = 110
         tableView.tableFooterView = nil
         navigationItem.title = "Filter"
+        navigationItem.rightBarButtonItem = UIBarButtonItem(image: UIImage(named: "info"), style: .done, target: self, action: #selector(PodcastsFilterViewController.viewInfo))
```

Comment:
I think there's a way to do this without bringing in an image

Reply:
see https://github.com/AlexRamey/CharlottesvilleArtfinder-iOS/blob/master/Charlottesville%20Artfinder/ARTDirectoryViewController.m